src/d_deh.c

```diff
@@ -1977,9 +1977,7 @@ void deh_procThing(DEHFILE *fpin, FILE* fpout, char *line)
                   if (fpout) fprintf(fpout, "Bits = 0x%08lX = %ld \n",
                                      value, value);
 
-                  if (value & MF_TRANSLUCENT ||
-                      mobjinfo[indexnum].flags & MF_TRANSLUCENT)
-                    D_SetTranslucency(indexnum);
+                  D_SetPredefinedTranslucency(indexnum);
```

Comment:
Okay, so as soon as the flags are changed, we consider translucency status as changed, right?

Reply:
Yes. I think it doesn't matter if the TRANSLUCENCY flag is enabled or disabled, it is still considered changed.

Reply:
Makes perfect sense!